lib/src/sql/value/value.rs

```diff
@@ -2938,7 +2938,7 @@ mod tests {
 		assert_eq!(24, std::mem::size_of::<crate::sql::table::Table>());
 		assert_eq!(56, std::mem::size_of::<crate::sql::thing::Thing>());
 		assert_eq!(48, std::mem::size_of::<crate::sql::model::Model>());
-		assert_eq!(16, std::mem::size_of::<crate::sql::regex::Regex>());
+		assert_eq!(32, std::mem::size_of::<crate::sql::regex::Regex>());
```

Comment:
The `Regex` struct size changed between version 1.8.0 and 1.9.0 so I changed the value in this test.